ios/FluentUI/Vnext/Button/Button.swift

```diff
@@ -128,6 +139,11 @@ struct FluentButtonBody: View {
             textColor = tokens.textColor.pressed
             borderColor = tokens.borderColor.pressed
             backgroundColor = tokens.backgroundColor.pressed
+        } else if isFocused {
+            iconColor = tokens.iconColor.hover
+            textColor = tokens.textColor.hover
+            borderColor = tokens.borderColor.hover
+            backgroundColor = tokens.backgroundColor.hover
```

Comment:
I think hover isn't quite different enough, it can be a bit hard still to tell if the focus moved or where it is.

Reply:
I think it's worth confirming these colors with the design team. Including the other question I asked about why just some styles that are getting focus colors.

Reply:
I've reached out, we'll see what they say

Reply:
They agreed with pressed for now, possibly revisit in the future.

Reply:
is there a github ticket for design team to make sure it is part of the fluent 2 design spec?